wikidict/lang/no/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""Norwegian language."""
+
+# Regex to find the pronunciation
+pronunciation = r""
+
+# Float number separator
+float_separator = ","
+
+# Thousads separator
+thousands_separator = " "
+
+# Markers for sections that contain interesting text to analyse.
+head_sections = ("norsk",)
+section_sublevels = (3, 4)
+etyl_section = ["Etymologi"]
+sections = (
+    *etyl_section,
+    "Substantiv",
+)
+
+# Templates to ignore: the text will be deleted.
+templates_ignored = ("",)
+
+# Templates more complex to manage.
+templates_multi = {
+    "": "",
+}
+
+# Release content on GitHub
+# https://github.com/BoboTiG/ebook-reader-dict/releases/tag/no
+release_description = """\
+Ord räknas: {words_count}
```

Comment:
@Duckbilled if you can provide a PR for the full translation, it would be awesome (you should use the english text as reference).